test/onnx/test_fx_to_onnx.py

```diff
@@ -470,52 +450,6 @@ def test_fake_tensor_mode_huggingface_open_llama(self):
             onnx.checker.check_model(export_output.model_proto)
             onnx.shape_inference.infer_shapes(export_output.model_proto)
 
-    def test_fake_tensor_mode_huggingface_google_t5(self):
```

Comment:
Have it in runtime

Reply:
I think you meant `moved this test to `test_fx_to_onnx_with_onnxruntime.py`... Please keep comments clear.